### PR TITLE
update activity to use set prefix

### PR DESCRIPTION
**Pull request by Codemonkey51**
why it always use ^, just fixed dat

### CODE REVIEW

**Review by jetstream0 — APPROVED** [review]
seems good

File: discordbot.py [hunk 1]
```diff
@@ -51,12 +51,13 @@ async def log_view(title, time):
 # 	channel = client.get_channel(770498997428551680)
 # 	await channel.send(title+" has been unlisted at "+time+" new url is "+newurl)
 
+prefix = config.get('prefix', '^')
 
 @client.event
-async def on_ready():
+async def on_ready():
```
Comment (mat-1) [inline discordbot.py:57]: Looks like you accidentally added an extra space lol

File: discordbot.py [hunk 2]
```diff
@@ -51,12 +51,13 @@ async def log_view(title, time):
...
 
+prefix = config.get('prefix', '^')
 
 @client.event
-async def on_ready():
+async def on_ready():	
 	print('ready')
 	await client.change_presence(
-		activity=discord.Game(name='^help')
+		activity= discord.Game(name= prefix + 'help')
```
Comment (mat-1) [inline discordbot.py:60]: The space after both equal signs shouldn't be there

**Review by mat-1 — CHANGES_REQUESTED** [review]

File: discordbot.py (hunk 2 above)
Comment (Codemonkey51) [reply discordbot.py:60]: Oh lol one sec

**Pull request merged**